src/routes/Profile/ProfilePageWalletConnected/ProfilePageWalletConnected.tsx

```diff
@@ -61,6 +64,8 @@ export const ProfilePageWalletConnected: React.FunctionComponent<ProfilePageWall
     onClaimBulkClick = doNothing,
     claimButtonModes = {},
     claimButtonBulkMode,
+    copyLinkButtonMode,
+    onCopyLinkButtonClick = doNothing,
```

Comment:
is "doNothing" something built in? or did we implement this?

Reply:
it is our implementation. You can check the imports. It kinda ensures if someone passes undefined that my code wont break